tests/mocks/test_account_linking_policy.py

```diff
@@ -0,0 +1,48 @@
+""""
+Integration tests for various pieces involved in external provider support.
+"""
+
+from unittest import TestCase, main
+from stormpath.resources import AccountLinkingPolicy
+
+try:
+    from mock import MagicMock
+except ImportError:
+    from unittest.mock import MagicMock
+
+
+class TestAccountLinkingPolicy(TestCase):
+
+    def test_modifying_account_linking_policy(self):
```

Comment:
@rdegges Should I change this? I've modelled it after an existing test which is very similar, but maybe that one was written before there were codacy checks so it remained intact.